content/en/docs/usage/certificate.md

```diff
@@ -97,79 +96,152 @@ The `Certificate` will be issued using the issuer named `ca-issuer` in the
 > issued X.509 certificates before the issue time to fix clock-skew issues,
 > leading to the working duration of a certificate to be less than the full
 > duration of the certificate. For example, Let's Encrypt sets it to be one hour
-> before issue time, so the actual *working duration* of the certificate is 89
-> days, 23 hours (the *full duration* remains 90 days).
+> before issue time, so the actual _working duration_ of the certificate is 89
+> days, 23 hours (the _full duration_ remains 90 days).
 
 A full list of the fields supported on the Certificate resource can be found in
-the [API reference documentation](../../reference/api-docs/#cert-manager.io/v1alpha2.CertificateSpec)
+the [API reference documentation](../../reference/api-docs/#cert-manager.io/v1alpha2.CertificateSpec).
 
-## Key Usages
+## X.509 key usages and extended key usages {#key-usages}
 
-cert-manager supports requesting certificates that have a number of custom key
-usages and extended key usages. Although cert-manager will attempt to honor this
-request, some issuers will remove, add defaults, or otherwise completely ignore
-the request and is determined on an issuer by issuer basis. The `CA` and
-`SelfSigned` `Issuer` will always return certificates matching the usages you have
-requested.
+cert-manager supports requesting certificates that have a number of [custom key
+usages](https://tools.ietf.org/html/rfc5280#section-4.2.1.3) and [extended key
+usages](https://tools.ietf.org/html/rfc5280#section-4.2.1.12). Although
+cert-manager will attempt to honor this request, some issuers will remove, add
+defaults, or otherwise completely ignore the request.
+The `CA` and `SelfSigned` `Issuer` will always return certificates matching the usages you have requested.
 
 Unless any number of usages has been set, cert-manager will set the default
-requested usages of "digital signature", "key encipherment", and "server auth".
+requested usages of `digital signature`, `key encipherment`, and `server auth`.
 cert-manager will not attempt to request a new certificate if the current
-certificate does not match the current key usages set.
+certificate does not match the current key usage set.
 
 An exhaustive list of supported key usages can be found in the [API reference
 documentation](../../reference/api-docs/#cert-manager.io/v1alpha2.KeyUsage).
 
-## Temporary Certificates whilst Issuing
+## Temporary Certificates while Issuing
```

Comment:
I just realized that this change is a breaking change for anyone who use the anchor ID, e.g.

> https://cert-manager.io/docs/usage/certificate/#temporary-certificates-whilst-issuing

Should I use instead:

```md
## Temporary Certificates while Issuing {#temporary-certificates-whilst-issuing}
```

to avoid breaking the anchor ID?